api/v1/controllers/bots.js

```diff
@@ -58,7 +59,44 @@ module.exports = {
    * @param {Function} next - The next middleware function in the stack
    */
   getBot(req, res, next) {
-    doGet(req, res, next, helper);
+    const resultObj = { reqStartTime: req.timestamp };
+
+    // try to get cached entry
+    redisCache.get(req.swagger.params.key.value, (cacheErr, reply) => {
+      if (reply) {
+        // reply is responsified bot object as string.
+        const botObject = JSON.parse(reply);
+
+        // add api links to the object and return response.
+        botObject.apiLinks = u.getApiLinks(
+          botObject.id, helper, req.method
+        );
+        return res.status(httpStatus.OK).json(botObject);
+      }
+      // if cache error, print error and continue to get bot from db.
+      if (cacheErr) {
+        console.log(cacheErr); // eslint-disable-line no-console
```

Comment:
isnt there some form of refocus logger for this ?

Reply:
no console logging from prod please--i know you just copied from lens implementation, but that actually predated our use of winston logger